database/postgresconn/postgresconn.go

```diff
@@ -275,7 +217,12 @@ func (p *Postgres) runStatement(statement []byte) error {
 	if strings.TrimSpace(query) == "" {
 		return nil
 	}
-	if _, err := p.conn.ExecContext(ctx, query); err != nil {
+	
+	if res, err := p.conn.ExecContext(ctx, query); err != nil {
+		fmt.Printf("%+v, err: %+v\n", res, err)
```

Comment:
should we take a proper logger instead of writing to STDOUT?

Reply:
I held off on that because this is a third party and introducing our logger seemed not quite right...